### PR TITLE
Swapping out build service from rollup to vue

### PR DESCRIPTION
**Pull request by ghost**
This is a pretty straightforward set of changes, with some side effects:

* Add `vue-cli-service` for building files. This should enable us to produce builds with less effort (see: the fact that rollup has a rather large config file, and vue-cli-service only has flags to replicate the same functionality)
* Edit `package.json` to swap out rollup-specific commands with vue-cli-service service commands

One side effect is that vue wants to name things according to type. I did manage to rename the library under the hood and keep the filename mostly sane, but instead of `vue-xivtooltips.min.js`, it's now `vue-xivtooltips.umd.min.js`. Merging this constitutes a breaking change for downstream consumers (of which I think there's only one), so we need to make sure they're using a pinned version in the repo.

I'm sure I'll get dinged on review for not removing rollup from the package.json, but I'll do that once I'm called out.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,10 @@
   "devDependencies": {
     "@rollup/plugin-commonjs": "^11.1.0",
     "@rollup/plugin-node-resolve": "^7.1.3",
+    "@vue/cli-plugin-babel": "^4.5.9",
+    "@vue/cli-plugin-eslint": "^4.5.9",
+    "@vue/cli-plugin-vuex": "^4.5.9",
+    "@vue/cli-service": "^4.5.9",
     "@vue/compiler-sfc": "^3.0.0-beta.9",
     "axios": "^0.19.2",
     "cross-env": "^7.0.2",
```
Comment (ayyaruq) [inline package.json]: You can kill this dependency.

File: package.json [hunk 2]
```diff
@@ -12,6 +12,10 @@
   "devDependencies": {
     "@rollup/plugin-commonjs": "^11.1.0",
     "@rollup/plugin-node-resolve": "^7.1.3",
```
Comment (ayyaruq) [inline package.json]: Ding. Also the other rollup plugins like commonjs etc. Also the rollup config.

**Review by ayyaruq — CHANGES_REQUESTED** [review]
Mostly ok, main thing I wonder about is if you'd like to support both web and nodejs usage, you'll prob need a vue.config.js config to tell babel to properly export stuff. Index.js seems to be handling it so maybe yolo, idk. Fix rollup+cross-env dependencies anyway.

File: package.json (hunk 1 above)
Comment (ghost) [reply package.json]: Removed.

**Review by ayyaruq — APPROVED** [review]

**Pull request merged**